CHANGELOG.md

```diff
@@ -16,6 +16,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Incorrect category_id input for test_category
 -   Assertion value for test_categories_limit_low and test_categories_limit_high
 
+## [2.21.1] - 2022-11-26
```

Comment:
We can leave the changes in `Unreleased` rather than create a new version just yet

Reply:
definitely, thanks👍🏻